frontend/public/components/monitoring/receiver-forms/slack-receiver-form.tsx

```diff
@@ -24,252 +35,228 @@ const GLOBAL_FIELDS = [
-          {t('public~Slack API URL')}
-        </label>
-        <Grid hasGutter>
-          <GridItem span={7}>
-            <span className="pf-v6-c-form-control">
-              <input
-                type="text"
-                id="slack-api-url"
-                aria-describedby="slack-api-url-help"
-                data-test-id="slack-api-url"
-                value={formValues.slack_api_url}
-                onChange={(e) =>
-                  dispatchFormChange({
-                    type: 'setFormValues',
-                    payload: { slack_api_url: e.target.value },
-                  })
-                }
-              />
-            </span>
-          </GridItem>
-          <GridItem span={1} /> {/* fixes an overlapping control issue */}
-          <GridItem span={4}>
-            <SaveAsDefaultCheckbox
-              formField="slackSaveAsDefault"
-              disabled={formValues.slack_api_url === globals?.slack_api_url}
-              label={t('public~Save as default Slack API URL')}
-              formValues={formValues}
-              dispatchFormChange={dispatchFormChange}
-              tooltip={t(
-                'public~Checking this box will write the API URL to the global section of the configuration file where it will become the default API URL for future Slack receivers.',
-              )}
+    <>
+      <Grid hasGutter>
+        <GridItem span={7}>
+          <FormGroup label={t('public~Slack API URL')} fieldId="slack-api-url" isRequired>
+            <TextInput
+              type="text"
+              id="slack-api-url"
+              data-test="slack-api-url"
+              value={formValues.slack_api_url}
+              onChange={(_e, value: string) =>
+                dispatchFormChange({
+                  type: 'setFormValues',
+                  payload: { slack_api_url: value },
+                })
+              }
             />
-          </GridItem>
-        </Grid>
-        <div className="help-block" id="slack-api-url-help">
-          {t('public~The URL of the Slack webhook.')}
-        </div>
-      </div>
-      <div className="form-group">
-        <label className="co-required" htmlFor="slack-channel">
-          {t('public~Channel')}
-        </label>
-        <span className="pf-v6-c-form-control">
-          <input
-            type="text"
-            id="slack-channel"
-            aria-describedby="slack-channel-help"
-            data-test-id="slack-channel"
-            value={formValues.slackChannel}
+            <FormHelperText>
+              <HelperText>
+                <HelperTextItem>{t('public~The URL of the Slack webhook.')}</HelperTextItem>
+              </HelperText>
+            </FormHelperText>
+          </FormGroup>
+        </GridItem>
+        <GridItem span={1} /> {/* fixes an overlapping control issue */}
+        <GridItem span={4}>
+          <SaveAsDefaultCheckbox
+            formField="slackSaveAsDefault"
```

Comment:
Aligns `checkbox` with 'input'
```suggestion
            className="pf-v6-u-align-content-center"
            formField="slackSaveAsDefault"
```